src/components/ProductLandingPage/Hero/Hero.scss

```diff
@@ -1,9 +1,10 @@
+// This is causing SWFW page's card container to have extra border line
 // Terraform Styling
-html[data-theme='dark'] {
-  .product-hero-container {
-    border-bottom: 1px solid var(--ifm-color-emphasis-100);
-  }
-}
+// html[data-theme='dark'] {
+//   .product-hero-container {
+//     border-bottom: 1px solid var(--ifm-color-emphasis-100);
+//   }
+// }
```

Comment:
When this is uncommented, you can override this in the SWFW CSS by being more specific with the CSS selectors. For example, our CSS might say something like:
```css
html[data-theme='dark'] {
  .swfw-landing-page {
    .product-hero-container {
      border-bottom: 0;
    }
  }
}
```